README.md

```diff
@@ -99,7 +99,7 @@ Then we pass it to TestSys which will create an EKS cluster, launch Bottlerocket
 
 **Caution**: The command below specifies `never` as the `--cluster-destruction-policy`.
 This is because creating a cluster takes a long time, and we might want to re-use it.
-To delete the cluster manually, use `eksctl delete cluster "external-cluster"` or delete the relevant Cloudformation stacks.
+To delete the cluster manually, use `eksctl delete cluster "external-cluster"` or delete the relevant CloudFormation stacks.
 You can also change the `--cluster-destruction-policy` to `onDelete` in the command below.
```

Comment:
from Matthias:
One minor thing I noticed when going through the quickstart myself: the suggestion to use "onDelete" for the cluster-destruction-policy should be "onDeletion"

Reply:
Will change that!